### DISCUSSION TITLE
Mentors?

### DISCUSSION THREAD

**Discussion by wmckeever** [codewars/codewars.com#2829] [General]
Hey I'm new to codewars and new to coding in general as I am transitioning careers. Does anyone know how I can go about finding a mentor? I'm trying to find someone who can help me understand understand the faults in my code and help me to learn and grow in this field. Any advice would be greatly appreciated.

**Comment by JEIPAO** [comment]
When you're available?

**Reply by Akinz-bobo** [reply]
Can I join the boat too? I have javascript experience but I am having problems solving algorithms. Please I'll appreciate any help

**Reply by wmckeever** [reply]
Hey thanks for the reply. I'm usually free on Fridays or anytime after 4pm since I am currently enrolled in a boot camp.

**Reply by Akinz-bobo** [reply]
4pm which time zone please and how do we meet?

**Comment by avatar8701** [comment]
I'm in the same boat. I came across this site. https://www.codementor.io/ which you might find helpful.

I haven't checked them out yet.

**Reply by wmckeever** [reply]
Thanks! I'm going to check it out!
